examples/integration/integration_test.go

```diff
@@ -48,6 +48,11 @@ func TestEcho(t *testing.T) {
 }
 
 func TestForwardResponseOption(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+		return
+	}
```

Comment:
Are these changes all necessary?

Reply:
Necessary, sort of? I think it's nice to have just in case people do want to skip the integration tests (this integration suite runs so fast it doesn't seem to actually matter). We had the pattern partially applied previously (and I missed some when I tried to bring it fully up to date).